docker/mantis/grafana/provisioning/dashboards/mantis-dashboard.json

```diff
@@ -1930,7 +1930,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "rate(app_consensus_blocks_generate_timer_seconds_sum[$__rate_interval])/rate(app_consensus_blocks_generate_timer_seconds_count[$__rate_interval])",
+              "expr": "rate(app_mining_blocks_generate_timer_seconds_sum[$__rate_interval])/rate(app_mining_blocks_generate_timer_seconds_count[$__rate_interval])",
```

Comment:
please remember to notify @nrdxp so that they have a chance to update the ops dashboards.

Reply:
I checked the dashboards on mantis-ops and they don't use this metric, but I did open a PR to fix the deployment https://github.com/input-output-hk/mantis-ops/pull/51